### DISCUSSION TITLE
GRID

### DISCUSSION THREAD

**Discussion by dpkhatua** [epochpic/epoch#505] [Ideas]
Hii Everyone,
Is there any way to create assymetric grids in the simulation window? For example, I want to make more grids at the center of the laser and lesser number of grids at other places.

Thanks.

**Comment by Status-Mirror** [comment]
Unfortunately not - EPOCH only supports cartesian grids. However, you could achieve the same computational speed-up effect by using fewer particles per cell further away from the focal spot. This is a technique I've seen done by [Vysckocil et al](https://iopscience.iop.org/article/10.1088/1361-6587/aab4c3/meta). See their Figure 2.

Cheers,
Stuart
